client/components/Message/Attachments/QuoteAttachment.tsx

```diff
@@ -67,7 +68,7 @@ export const QuoteAttachment: FC<QuoteAttachmentProps> = ({
 					<MarkdownText variant='inline' content={text} />
 					{attachments && (
 						<Attachment.Inner>
```

Comment:
I think that instead of checking for a prop, you could add a negative margin in this component. Don't forget that you might need to decrease the margin by another 4px to counteract the childrens 4 pixel margin

Reply:
fair enough!